doc/source/conf.py

```diff
@@ -67,8 +68,7 @@
     "use_meilisearch": {
         "api_key": os.getenv("MEILISEARCH_API_KEY", ""),
         "index_uids": {
-            "ansys-ansys-sphinx-theme-sphinx-docs": "ansys-sphinx-theme",
-            "ansys-pyaedt-sphinx-docs": "PyAEDT",
+            f"ansys-sphinx-theme-{get_version_match(__version__)}": "ansys-sphinx-theme",
```

Comment:
Maybe it is better to add the `v*`:

```suggestion
            f"ansys-sphinx-theme-v{get_version_match(__version__)}": "ansys-sphinx-theme",
```

Reply:
This should also be updated in the CI files generating the index.

Reply:
Agreed.